frontend/ui/src/widgets/property-sheet/color-picker.tsx

```
@@ -0,0 +1,202 @@
+
+      <div className="flex justify-between">
+        {colorGroups.map((group, groupIndex) => (
+          <div key={`group-${groupIndex}`} className="flex flex-col gap-1">
+            {group.map((color, colorIndex) => (
+              <button
+                key={`color-${groupIndex}-${colorIndex}`}
+                className="h-6 w-6 cursor-pointer rounded-sm transition-transform hover:scale-110"
+                style={{ backgroundColor: color }}
+                onClick={() => onColorSelect({ hex: color })}
+                title={color}
+              />
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)